data/src/split.py

```diff
@@ -44,7 +44,84 @@ def _detect_best_script_name(
         # The script names in Unicode data tables have underscores instead of
         # whitespace to enable parsing. See:
         #   https://www.unicode.org/Public/13.0.0/ucd/Scripts.txt
-        return script_probs[0][0].replace("_", " "), script_probs[0][1]
+        return script_probs[0][0], script_probs[0][1]
+
+
+def _remove_duplicates(script_dict: Dict[str, str]) -> Dict[str, str]:
```

Comment:
This function has been repeated from `languages_update.py`. Is that intended?

Reply:
I think someone mentioned this but `_remove_duplicates()` would function better as a test in `test_data` that enforces our ISO 15924 script key convention in `languages.json`.